resources/puppetlabs/lein-ezbake/staging-templates/project_data.yaml.mustache

```diff
@@ -45,7 +43,6 @@ templates:
   - ext/redhat/init.erb
   - ext/redhat/init.suse.erb
   - ext/debian/changelog.erb
-  - ext/debian/control.erb
   - ext/debian/dirs.erb
```

Comment:
mm, we might be able to get rid of a few more of these deb files, lemme check

Reply:
yeah, I think there's a lot more to get rid of, but this is what I got to testing last week, so I wanted to get it in before pivoting to higher-priority stuff.
If there are files in that list that are already gone or things you're pretty confident can go, I'll add that to this PR, but otherwise, I think this work is less important than some of the other things we have in flight.

Reply:
Yeah, there are a few very related files (in the same directory, the other debian build files that fpm doesn't use). It seems like if we don't clean those now we will never do it, i'm running a build now just to test.

Reply:
okay cool, yeah, related files seem like a good thing to include. thanks for testing that!